scripts/dts/extract/flash.py

```diff
@@ -10,6 +10,11 @@
 from extract.default import default
 from extract.reg import reg
 
+flash_dev_label_prefix = {
```

Comment:
We should remove this, we don't need to generate it, just generate the flash controller label name for each area.